### PR TITLE
fixed error in template_basic_plugin pyproject.toml

### PR DESCRIPTION
**Pull request by evrose54**
# Related Issues
fixes NRLMMD-GEOIPS/template_basic_plugin#20

# Summary
Fixed a small bug in template_basic_plugin's pyproject.toml. The file was incorrectly set up after the switch to poetry and my vscode TOML highlighter caught the error with the dependency that I fixed in this PR. I'm surprised we were able to install this package, TBH. Since it's a two line fix I haven't added release notes, nor do I think this repo is set up for brassy yet.

@mindyls if you can inform me of how to set up brassy for this repo so I can create the release notes for this PR I'd be happy to!

### CODE REVIEW

**Review by jsolbrig — APPROVED** [review]

**Comment by biosafetylvl5** [comment]
Related to https://github.com/NRLMMD-GEOIPS/geoips/issues/666

**Comment by biosafetylvl5** [comment]
The build docs CI check is expected for now, see related issue #28

Also, this PR fixes #24

File: docs/source/releases/latest/t [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Add a ``latest`` directory and add a yaml release note.
+
+::
+
+    added: docs/source/release/latest
+    added: docs/source/release/latest/20-update-pyproject-depedencies.yaml
+    deleted: 
+    modified: 
+    moved:
```
Comment (mindyls) [inline docs/source/releases/latest/t:37]: @evrose54 Should this file be dleeted?

Comment (jsolbrig) [reply docs/source/releases/latest/t:37]: Plese!

File: docs/source/releases/latest/20-update-pyproject-depedencies.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    modified:
+    - tests/scripts/amsr2.global_clean.89-PCT-Using-Product-Defaults.sh
+    - tests/scripts/amsr2.tc_clean.89-PCT-Fully-Specified.sh
+    - tests/scripts/test_config.sh
+    moved:
+    - ''
+  related-issue:
+    number: 27
+    repo_url: ''
+  title: 'Set up repo for release notes built via brassy'
```
Comment (mindyls) [inline docs/source/releases/latest/20-update-pyproject-depedencies.yaml:34]: I think this section can/should be removed.  I don't think release notes need to reference themselves.  No more release notes in release notes!

Comment (mindyls) [reply docs/source/releases/latest/20-update-pyproject-depedencies.yaml:34]: Actually, it should be updated so it is ONLY the testing updates (ie, remove all the stuff about release notes, and make that section just that the test scripts were updated for "geoips run single_source")

File: docs/source/releases/latest/t (hunk 1 above)
Comment (evrose54) [reply docs/source/releases/latest/t:37]: _plese_. I didn't add this and was confused too

Comment (mindyls) [reply docs/source/releases/latest/t:37]: @biosafetylvl5

Comment (biosafetylvl5) [reply docs/source/releases/latest/t]: Plese 😭 Deleted now

File: docs/source/releases/latest/20-update-pyproject-depedencies.yaml (hunk 2 above)
Comment (biosafetylvl5) [reply docs/source/releases/latest/20-update-pyproject-depedencies.yaml:34]: I will change this!

**Review by jsolbrig — APPROVED** [review]

File: docs/source/releases/latest/20-update-pyproject-depedencies.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    modified:
+    - 'pyproject.toml'
+    moved:
+    - ''
+  related-issue:
+    number: 20
+    repo_url: ''
+  title: 'Fixed error in template_basic_plugin pyproject.toml'
+    repo_url: ''
+  title: 'Set up repo for release notes built via brassy'
```
Comment (mindyls) [inline docs/source/releases/latest/20-update-pyproject-depedencies.yaml:18]: Do we want to reference the test script updates in here?

File: docs/source/releases/latest/20-update-pyproject-depedencies.yaml (hunk 2 above)
Comment (biosafetylvl5) [reply docs/source/releases/latest/20-update-pyproject-depedencies.yaml]: Fixed!

**Review by mindyls — DISMISSED** [review]
LOOKS GREAT.

File: docs/source/releases/latest/20-update-pyproject-depedencies.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Refactoring Updates:
+  - description: |
+      Replaced all occurences of ``run_procflow`` in the test scripts with ``geoips run``
+      so the files don't conflict with the content in 2024 tutorial.
+    files:
+      modified:
+        - tests/scripts/amsr2.global_clean.89-PCT-Using-Product-Defaults.sh
+        - tests/scripts/amsr2.tc_clean.89-PCT-Fully-Specified.sh
+        - tests/scripts/test_config.sh
+    title: 'Replaced all occurences of ``run_procflow`` in test scripts with ``geoips run``'
```
Comment (mindyls) [inline docs/source/releases/latest/20-update-pyproject-depedencies.yaml:28]: image

Must be a typo in YAML release note?

Comment (evrose54) [reply docs/source/releases/latest/20-update-pyproject-depedencies.yaml:28]: This should be fixed now. I hate quotes.

**Review by jsolbrig — APPROVED** [review]

**Review by mindyls — APPROVED** [review]
This is expected,  my_package package name does not equal template basic plugin repo name.  Will fix later.

**Pull request merged**